site/en/adminGuide/upgrade-helm.md

```diff
@@ -0,0 +1,85 @@
+---
+id: upgrade-helm.md
+label: Helm
+order: 1
+group: upgrade.md
+summary: Learn how to upgrade Milvus v2.1 to v2.2 with Helm.
+---
+
+{{tab}}
+
+# Upgrade Milvus with Helm
+A major change in Milvus 2.2 is the meta structure of segment indexes. This topic describes how to use Helm to migrate the meta and upgrade Milvus from v2.1.x to v2.2.0.
+
+<div class="alert note">
+This script only applies to Milvus installed on a K8s cluster. Rollback to the previous version with the <code>rollback</code> operation first if an error occurs during an upgrade.
+</div>
+
+## Parameters
+
+| Parameters   | Description                                                      | Default value                    | Required                |
+| ------------ | ---------------------------------------------------------------- | ---------------------------- | ----------------------- |
+| `i`          | The Milvus instance name.                                 | `None`                         | True                    |
+| `n`          | The namespace that Milvus is installed in.                | `default`                      | False                   |
+| `s`          | The source Milvus version.                                | `None`                         | True                    |
+| `t`          | The target Milvus version.                               | `None`                         | True                    |
+| `r`          | The root path of Milvus meta.                             | `by-dev`                       | False                   |
+| `w`          | The new Milvus image tag.                                 | `milvusdb/milvus:v2.2.0`       | False                   |
+| `m`          | The meta migration image tag.                             | `harbor.milvus.io/milvus/meta-migration:20221025-e54b6181b`       | False                   |
+| `o`          | The meta migration operation.                             | `migrate`                      | False                   |
+| `d`          | Whether to delete migration pod after the migration is completed.          | `false`                        | False                   |
+
+## Migrate meta
```

Comment:
Here user may be confused with these descriptions. User may thought that he will take up the following steps. Those steps just shows how the script will do for meta migration.